dev/tests/integration/testsuite/Magento/MediaStorage/Helper/File/Storage/DatabaseTest.php

```diff
@@ -52,7 +52,7 @@ protected function setUp(): void
     /**
      * test for \Magento\MediaStorage\Model\File\Storage\Database::deleteFolder()
      *
-     * @magentoDbIsolation enabled
+     * @magentoDbIsolation disabled
```

Comment:
Note for QA: Need to make sure this test is not leaving any data in DB after test run.